data/json/items/gun/9mm.json

```
@@ -658,7 +658,32 @@
     "color": "dark_gray",
     "ammo": "9mm",
     "magazine_well": "250 ml",
-    "magazines": [ [ "9mm", [ "glockmag", "glockbigmag", "glock17_17", "glock17_22", "glock_drum_50rd", "glock_drum_100rd" ] ] ],
+    "magazines": [ [ "9mm", [ "glockmag", "glockbigmag", "glock17_17", "glock17_22", "glock_drum_50rd", "glock_drum_100rd", "kriss_vector9_mag" ] ] ],
     "modes": [ [ "DEFAULT", "semi", 1 ], [ "AUTO", "auto", 3 ] ]
+  },
+  {
+    "id": "kriss_vector9",
+    "copy-from": "smg_base",
+    "type": "GUN",
+    "looks_like": "TDI",
+    "name": { "str": "Kriss Vector 9mm" },
+    "description": "A submachine gun with a sleek in-line design. It features a 2-round burst mode to make the least out of its unique recoil mitigation system alongside a blistering full-auto mode. This one is chambered in 9mm and uses full-size Glock 17 magazines.", //G19 mags are too short to fit in the Kriss.
```

This attempt at a comment completely bricks the formatting, remove it.
```suggestion
    "description": "A submachine gun with a sleek in-line design. It features a 2-round burst mode to make the least out of its unique recoil mitigation system alongside a blistering full-auto mode. This one is chambered in 9mm and uses full-size Glock 17 magazines.",
```